src/main_menu.cpp

```diff
@@ -82,8 +109,9 @@ void main_menu::print_menu_items( const catacurses::window &w_in,
         }
     }
 
-    if( utf8_width( remove_color_tags( text ) ) > getmaxx( w_in ) ) {
-        offset.y -= std::ceil( utf8_width( remove_color_tags( text ) ) / getmaxx( w_in ) );
+    int text_width = utf8_width_notags( text.c_str() );
+    if( text_width > getmaxx( w_in ) ) {
+        offset.y -= std::ceil( text_width / getmaxx( w_in ) );
```

Comment:
Sorry, I thought there were only color tags when I made that change. And maybe I'm a bit late, but here the text should already be stripped of the shortcut tag by the `shortcut_text` call above, so the change is not necessary.